src/app/shared/services/auth.service.ts

```diff
@@ -67,7 +68,9 @@ export class AuthService {
       displayName: user.displayName ? user.displayName : user.email
     };
 
-    if (user.photoURL) data.photoUrl = user.photoURL;
+    const defaultPhotoURL = "https://openclipart.org/image/2400px/svg_to_png/277084/Male-Avatar-3.png";
```

Comment:
Would be better to store the url constant in a sepparate file for url constants.
The file should be `app/shared/constants/urls.ts` and to have structure something like this:
`
export const urls = {
  defaultPhoto: 'https://content.recklessnetwork.com/monthly_2016_08/dogavatar3.jpg.0bea41affc676bce724cb2c9f5ed9f15.thumb.jpg.ff828b4ca69782411f405ab1c11493ba.jpg'
}
`

I changed the url because that photo is too big, and takes a long time to load for a simple avatar.
Also, please use ' instead of " inside .ts files (and " inside .html and .less files)

Reply:
Fixed